### ISSUE TITLE
stderr is not propagated

### ISSUE THREAD

**Issue by afriza** [shellhub-io/shellhub#1969]
via shellhub:
```console
$ ssh user@device@shellhub sh -c " ' echo StandardOutput ; echo StandardError >&2 ' "
StandardOutput
```

direct:
```console
$ ssh user@device sh -c " ' echo StandardOutput ; echo StandardError >&2 ' "
StandardOutput
StandardError
```

I'm still using v0.9.0 ... not sure if recent changes already support this ...

**Comment by otavio** [comment]
It'd be nice if you could test the last RC and see if it helps.

**Comment by afriza** [comment]
do I need to update the server and agent or only agent is sufficient?

**Comment by otavio** [comment]
Yes. The fix requires the server updated.

**Comment by afriza** [comment]
hm.. if we update the server; and some agents are not presently online, there is possibility of version mismatch because we cannot update all agents at the same time.. would old agents be able to connect?

**Comment by otavio** [comment]
I'd prefer if you do a test. Not upgrade your server yet.

**Comment by gustavosbarreto** [comment]
Maybe this has been fixed already by #1968. I'll confirm this.

**Comment by afriza** [comment]
I haven't tried the new code.. but I have a feeling you need to propagate stderr for "exec" request type as well..

https://stackoverflow.com/q/6770206/109747
